semantic_release/vcs_helpers.py

```diff
@@ -25,25 +31,34 @@ def get_commit_log(from_rev=None):
 
 def get_last_version(skip_tags=None):
     """
-    return last version from repo tags
+    Return last version from repo tags.
 
-    :return: a string contains version number
+    :return: A string contains version number.
     """
+
+    _check_repo()
     skip_tags = skip_tags or []
 
-    def version_finder(x):
-        if isinstance(x.commit, TagObject):
-            return x.tag.tagged_date
-        return x.commit.committed_date
+    def _version_finder(tag):
```

Comment:
Why does this function name start with an `_`?

Reply:
It is intended to be a private function, not part of the public API
https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references

For private functions no docstring is required...

Reply:
Please remove the `_`.

Reply:
As you wish... done!